appservice/src/main/java/nu/yona/server/appresources/LocalizationFilter.java

```diff
@@ -0,0 +1,118 @@
+package nu.yona.server.appresources;
+
+import java.io.IOException;
+import java.util.Locale;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletRequestWrapper;
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.stereotype.Component;
+
+import nu.yona.server.Translator;
+import nu.yona.server.properties.YonaProperties;
+
+@Component
```

Comment:
I'll add that comment.
The ``LocaleChangeInterceptor`` sets the locale based on a query parameter, thus allowing to influence the request through the URL. That's very different from what the ``LocalizationFilter`` does.